### PR TITLE
fix(amazonq): update login screen and auth notification

### PR DESCRIPTION
**Pull request by ivikash**
- Update Styling for Login Page and Reauth page
- Show notification once if a user is not logged in when Amazon Q extension activates.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ivikash** [comment]
Screenshot 2024-06-18 at 4 17 05 PM
Screenshot 2024-06-18 at 4 16 59 PM
Screenshot 2024-06-18 at 4 16 37 PM
Screenshot 2024-06-18 at 4 16 13 PM
Screenshot 2024-06-18 at 4 16 04 PM
Screenshot 2024-06-18 at 4 15 51 PM
Screenshot 2024-06-18 at 4 15 42 PM

**Review by justinmk3 — APPROVED** [review]
Should this wait until the "split sessions" branch is merged?

**Comment by ivikash** [comment]
> Should this wait until the "split sessions" branch is merged?
@justinmk3 we can still go ahead and merge this. The CSS only impacts Amazon Q extension and not the toolkits :)

**Comment by hayemaxi** [comment]
Awesome! Let's add a changelog item for amazon Q, then lgtm!

File: packages/amazonq/.changes/next-release/Bug Fix-9130bd3b-edc4-4749-8c96-bd9a4fb9f03c.json
Comment (hayemaxi) [inline packages/amazonq/.changes/next-release/Bug Fix-9130bd3b-edc4-4749-8c96-bd9a4fb9f03c.json:3]:
```suggestion
	"type": "Feature",
	"description": "UX: New style for the login window"
```

**Pull request merged**
